conn.go

```diff
@@ -479,6 +496,24 @@ func (c *Conn) Send(destination, contentType string, body []byte, opts ...func(*
 	return nil
 }
 
+// SendWithReplyDestination is the same as Send except it automatically adds in the 'reply-to' header
+// to the *frame.Frame pointer that is passed in. This will allow any consumer to know where to send
+// a reply to, the broker will generally translate that location to a dynamic address.
+func (c *Conn) SendWithReplyDestination(destination, replyDestination, contentType string, body []byte, opts ...func(*frame.Frame) error) error {
```

Comment:
I'm hesitant to add this option, since the `reply-to` header is specific to RabbitMQ (and we're trying to be broker agnostic) and it's really easy for consumers to implement using the `Send` method

Reply:
Yeah, no problem, I am just a fan of abstraction. It's just a convenience method. It's not important. I will remove it.